smart-frontend/app/src/App.tsx

```diff
@@ -17,39 +17,34 @@
  */
 import React from 'react';
 import { Provider } from 'react-redux';
-import { BrowserRouter, Route, Routes } from 'react-router-dom';
+import { BrowserRouter, Outlet, Route, Routes } from 'react-router-dom';
 import { store } from '@store';
-import { Button, IconButton, Input } from '@uikit';
 import './styles/app.scss';
+import MainLayout from '@layouts/MainLayout/MainLayout';
+import ClusterInfoPage from '@pages/ClusterInfoPage/ClusterInfoPage';
+import RulesPage from '@pages/RulesPage/RulesPage';
+import ActionsPage from '@pages/ActionsPage/ActionsPage';
+import AuditPage from '@pages/AuditPage/AuditPage';
 
 function App() {
   return (
     <BrowserRouter>
       <Provider store={store}>
         <Routes>
+          {/*<Route path="/login" element={<LoginPage />} />*/}
```

Comment:
do we still need this?

Reply:
it's not forgotten comment. It's a blueprint for future changes